src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -1,197 +1,95 @@
-# Become an Amplifier Verifier (Worker)
+# Become an Amplifier Verifier
 
 import { Callout } from "/src/components/callout"
 
 <Callout>
```

Comment:
@ffe9f8 , can you include this same callout on all of the `introduction.mdx`, `introduction.mdx`, and `gmp.mdx` pages?